packages/styled-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "styled-components",
-  "version": "5.0.0-beta.5",
+  "version": "5.0.0-beta.6",
```

Comment:
Accidental bump in a PR?

Reply:
Just needs a rebase - it's a stray commit but one that's already in the canary branch 🤷‍♂